src/arvcamera.h

```diff
@@ -141,8 +141,8 @@ ARV_API void		arv_camera_set_exposure_mode		(ArvCamera *camera, ArvExposureMode
 
 ARV_API gboolean	arv_camera_is_gain_available		(ArvCamera *camera, GError **error);
 ARV_API gboolean	arv_camera_is_gain_auto_available	(ArvCamera *camera, GError **error);
-ARV_API void		arv_camera_select_gain				(ArvCamera *camera, const char *selector, GError **error);
```

Comment:
Please keep selector as the parameter name, and fix the c file

Reply:
Done.